coding-standards.md

```diff
@@ -0,0 +1,101 @@
+# Coding standards for potential open source contributors
```

Comment:
Maybe rename this file to `CONTRIBUTING.md` which is a standard filename to guide contributors.

https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors#adding-a-contributing-file